NewForce-at-Mountwest/reactive-nutshell-twofoot
# Cf branch

tasks and auth should be fully functional

to test:
-user should be taken to login(only registration atm) on default, and all nav links should redirect to login if 
 user has not logged in
-credentials should be the new user's id
-upon login should redirect to home(news or "/")
-tasks should have an add new task button which redirects to a form to create a new task
-when a task is added, it should be added to the main tasks view ("/tasks")
-tasks should have an edit and delete button, alongside a checkbox that when clicked, will ARCHIVE the 
 task
-archived tasks should remain in database, with isCompleted being changed to true
-completed tasks should not appear in the tasks list
-clicking delete should remove a task and refresh the main tasks view
-edit should open a new form that autopopulates with the previous values for that task
-when user clicks save on the edit page, it should go back to the main tasks few and display changes
